libs/core/tests/benchmarks/test_async_callbacks.py

```diff
@@ -0,0 +1,52 @@
+# ruff: noqa: ARG002
```

Comment:
It would be better to escape ruff rule on individual lines, not for the whole file.